### PR TITLE
CASMTRIAGE-6625: Add known issue for spire job in install

### PR DESCRIPTION
**Pull request by ndavidson-hpe**
# Description

Adds in a known issue where the cray-spire-update-bss job is deleted after some time. The workaround is to recreate the job from the helm chart and restart to ensure the steps are correct.\

# Checklist

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: install/install_csm_services.md
Comment (rustydb) [inline install/install_csm_services.md:1]: The indentation breaks the ordered list and formatting.

image

For anything that's a part of a list item, it should be indented to at least the first character in the first line.

e.g.

````markdown
1. Lorem ipsom

   Another statement, aligned with "L"

1. Another step

   ```bash
   echo "hello world"
   ```

````

**Review by rustydb — CHANGES_REQUESTED** [review]

File: install/install_csm_services.md
Comment (ndavidson-hpe) [reply install/install_csm_services.md:1]: Will fix.

File: install/install_csm_services.md [hunk 1]
```diff
@@ -218,6 +218,39 @@ The following error may occur during the `Deploy CSM Applications and Services`
...
+The Following error may occur during `Create base BSS global boot parameters` step:
+
+```text
+kubectl -n spire get "${SPIRE_JOB}" -o json | jq 'del(.spec.selector)' \
+>     | jq 'del(.spec.template.metadata.labels."controller-uid")' \
+>     | kubectl replace --force -f -
+error: the server doesn't have a resource type ""
+No resources found
+error: no objects passed to replace
+```
```
Comment (rustydb) [inline install/install_csm_services.md]:
````suggestion
     The Following error may occur during `Create base BSS global boot parameters` step:

     ```text
     kubectl -n spire get "${SPIRE_JOB}" -o json | jq 'del(.spec.selector)' \
     >     | jq 'del(.spec.template.metadata.labels."controller-uid")' \
     >     | kubectl replace --force -f -
     error: the server doesn't have a resource type ""
     No resources found
     error: no objects passed to replace
     ```
````

File: install/install_csm_services.md [hunk 2]
```diff
@@ -218,6 +218,39 @@ The following error may occur during the `Deploy CSM Applications and Services`
...
+     The Following error may occur during `Create base BSS global boot parameters` step:
+
+     ```text
+     kubectl -n spire get "${SPIRE_JOB}" -o json | jq 'del(.spec.selector)' \
+     >     | jq 'del(.spec.template.metadata.labels."controller-uid")' \
+     >     | kubectl replace --force -f -
+     error: the server doesn't have a resource type ""
+     No resources found
+     error: no objects passed to replace
+     ```
```
Comment (rustydb) [inline install/install_csm_services.md]:
````suggestion
   The Following error may occur during `Create base BSS global boot parameters` step:

   ```text
   kubectl -n spire get "${SPIRE_JOB}" -o json | jq 'del(.spec.selector)' \
   >     | jq 'del(.spec.template.metadata.labels."controller-uid")' \
   >     | kubectl replace --force -f -
   error: the server doesn't have a resource type ""
   No resources found
   error: no objects passed to replace
   ```
````

**Review by rustydb — APPROVED** [review]

**Review by heemstra — APPROVED** [review]

**Pull request merged**